### PR TITLE
Ignore UNCONFIRMED if set

### PR DESCRIPTION
**Pull request by gizzmojr**
Per Product setting from bz administration

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -154,6 +154,11 @@
             function loadBoard() {
                 clearBoard();
                 startSpinner();
+                doGET("/rest/product/" + bzProduct, function(response) {
+                    if (!(response.products[0].has_unconfirmed)) {
+                        document.getElementsByClassName("board-column").UNCONFIRMED.hidden = true;
+                    }
+                });
```
Comment (leif81) [inline index.html]: I feel like there may be a race condition here? Maybe we should do this after the board columns get added below.

**Comment by gizzmojr** [comment]
Ok fixed up, moved it from a bad spot in a loop. 

Don't thinking putting it before loadBugs within the other doGET is a good idea if that's what you're thinking.

**Comment by leif81** [comment]
Slightly better...but still a race condition. :car: :racehorse: 

From what I can tell, this and `loadBugs` have the same requirement. Which is they both need the board columns to exist for the rest of the logic to work as expected.

**Comment by gizzmojr** [comment]
Are you running the code? It shows unconfirmed on the screen before it gets disabled?

**Comment by leif81** [comment]
Ya I see a flicker of the board as the column gets removed.

**Comment by leif81** [comment]
:ship:

**Comment by gizzmojr** [comment]
:+1: 
These are fun
:hamburger:

**Pull request merged**